ape_etherscan/utils.py

```diff
@@ -3,6 +3,8 @@
     "ethereum": "ETHERSCAN_API_KEY",
     "fantom": "FTMSCAN_API_KEY",
     "optimism": "OPTIMISTIC_ETHERSCAN_API_KEY",
+    "polygon-zkevm": "POLYGON_ZKEVM_API_KEY",
+    "base": "BASE_API_KEY",
```

Comment:
```suggestion
    "base": "BASESCAN_API_KEY",
```

just basing these off the web service names, like all the others